jupyter_releaser/lib.py

```diff
@@ -116,7 +116,7 @@ def draft_changelog(
     body = title
 
     if npm_versions:
-        body += f"```\n{npm_versions}\n```"
+        body += f"\n```{npm_versions}\n```"
```

Comment:
Leading `\n` are already being added in:

https://github.com/jupyter-server/jupyter_releaser/blob/ebe525abd9c73ad57c897426b38ae756688ce65c/jupyter_releaser/npm.py#L147-L155